src/utils/discordHook.ts

```diff
@@ -0,0 +1,35 @@
+import properties from '@src/utils/properties';
+import { Webhook } from 'discord-webhook-node';
+
+interface DiscordManager {
+    getReportReviewHook(): Webhook | undefined;
+    getServerStatusHook(): Webhook | undefined;
+}
+
+class DiscordManagerImpl implements DiscordManager {
+    getReportReviewHook(): Webhook | undefined {
+        const hook = new Webhook(
+            'https://discord.com/api/webhooks/957212284617375744/hQOZd1qGPnv35YuhCXTHJB9ZtlRifhigpjXup8nbYiivX3PkJVaNePH-IDnC9zDuC4Z0'
```

Comment:
🔧 이 부분을 노출시키지 않으려고 DISCORD_URL_FOR_SERVER_STATUS를 정의하신것 아닌가요?
```suggestion
            process.env.DISCORD_URL_FOR_SERVER_STATUS
```

Reply:
저 url과 DISCORD_URL_FOR_SERVER_STATUS은 다른 URL입니다!
reportRevieHook:  시향노트 신고 채널
serverStatusHook: 서버 모니터링 채널

serverStatusHook 가 아닌 시향노트 신고 채널은 기존 방식 유지 했습니다. 
저부분도 향후 업데이트 하면 좋을 거 같아요.
(단 그전에 디스코트 훅이 여러개가 될 텐데 어떻게 관리할지 고민해보면 좋을 거 같아요.)

Reply:
해당 값을 DISCORD_HOOK_FOR_REPORT_REVIEW로 분리한 커밋 추가 했습니다.